src/gui/torrentcontentmodel.cpp

```diff
@@ -91,14 +91,18 @@ namespace
         QIcon icon(const QFileInfo &info) const override
         {
             SHFILEINFO sfi = { 0 };
-            HRESULT hr = ::SHGetFileInfoW(info.absoluteFilePath().toStdWString().c_str(),
+            QString nativePath = Utils::Fs::toNativePath(info.absoluteFilePath());
```

Comment:
Have you seen the discussion about QPixmapCache here? -> https://github.com/qbittorrent/qBittorrent/pull/7253#issuecomment-321893977
Would that work?